### ISSUE TITLE
✅ 🐞 Bug Report: Gallery Pro Uses Elementor Lightbox Even When Disabled! - [ED-10590]

### ISSUE THREAD

**Issue by black-eye** [elementor/elementor#11193]
## Prerequisites

- [x] I have searched for similar issues in both open and closed tickets and cannot find a duplicate.
- [x] The issue still exists against the latest stable version of Elementor.

## Description

It's simple as discribed in the title: when using Gallery Pro, the new lightbox is used, even when I disabled it. 

Example page: https://www.blackeye.cz/report/gallery-test/. It shows different types of images, all of them use external lightbox, except Gallery Pro, which uses Elementor Lightbox.

Settings:

![obrazek](https://user-images.githubusercontent.com/5634870/79878549-d8711980-83ed-11ea-9d40-73e1bffe5863.png)

## Steps to reproduce

1. Disable Elementor Lightbox
2. Add Galery Pro
3. Click on the thumbnail and enjoy the lightbox you have turned off ;-)

## Isolating the problem

- [x] This bug happens with only Elementor plugin active (and Elementor Pro).
- [ ] This bug happens with a default WordPress theme active.
- [x] I can reproduce this bug consistently using the steps above.

## Environment

<details>
<summary>System Info</summary>
```
					
== Server Environment ==
	Operating System: Linux
	Software: Apache/2.4.10 (Debian)
	MySQL version: (Debian) v5.5.62
	PHP Version: 7.2.8-1+0~20180725113345.2+jessie~1.gbp571e56
	PHP Max Input Vars: 1000
	PHP Max Post Size: 8M
	GD Installed: Yes
	ZIP Installed: Yes
	Write Permissions: All right
	Elementor Library: Connected

== WordPress Environment ==
	Version: 5.3.2
	Site URL: https://www.blackeye.cz/report
	Home URL: https://www.blackeye.cz/report
	WP Multisite: No
	Max Upload Size: 8 MB
	Memory limit: 256M
	Permalink Structure: /%postname%/
	Language: cs
	Timezone: Europe/Prague
	Debug Mode: Active

== Theme ==
	Name: re-port WPBF
	Version: 1.0.0
	Author: Petr Jandák
	Child Theme: Yes
	Parent Theme Name: Page Builder Framework
	Parent Theme Version: 2.3.1
	Parent Theme Author: David Vongries

== User ==
	Role: administrator
	WP Profile lang: cs_CZ
	User Agent: Mozilla/5.0 (Windows NT 10.0; Win64; x64; rv:75.0) Gecko/20100101 Firefox/75.0

== Active Plugins ==
	Activity Log
		Version: 2.5.2
		Author: Activity Log Team

	Admin Menu Editor
		Version: 1.9.5
		Author: Janis Elsts

	Caldera Forms
		Version: 1.8.11
		Author: Caldera Labs

	Central Color Palette
		Version: 1.14.6
		Author: Daniel Menzies

	Classic Editor
		Version: 1.5
		Author: WordPress Contributors

	Clean Image Filenames
		Version: 1.2.1
		Author: Upperdog

	Disable Comments
		Version: 1.10.2
		Author: Samir Shah

	Duplicate Post
		Version: 3.2.4
		Author: Enrico Battocchi

	Elementor
		Version: 2.9.7
		Author: Elementor.com

	Elementor Extras
		Version: 2.2.27
		Author: Namogo

	Elementor Pro
		Version: 2.9.3
		Author: Elementor.com

	Intuitive Custom Post Order
		Version: 3.1.2
		Author: hijiri

	iThemes Security
		Version: 7.7.1
		Author: iThemes

	Lightbox with PhotoSwipe
		Version: 2.92
		Author: Arno Welzel

	Members
		Version: 3.0.4
		Author: MemberPress

	Nested Pages
		Version: 3.1.9
		Author: Kyle Phillips

	Page Builder Framework Premium Addon
		Version: 2.3.1
		Author: David Vongries

	Rank Math SEO
		Version: 1.0.42.3
		Author: Rank Math

	ShortPixel Image Optimizer
		Version: 4.17.3
		Author: ShortPixel

	Simple History
		Version: 2.33.2
		Author: Pär Thernström

	SVG Support
		Version: 2.3.18
		Author: Benbodhi

	TinyMCE Advanced
		Version: 5.3.0
		Author: Andrew Ozz

	WP Media folder
		Version: 5.1.2
		Author: Joomunited

	WPvivid Backup Plugin
		Version: 0.9.38
		Author: WPvivid Team

	WPvivid Backup Pro
		Version: 1.9.13
		Author: wpvivid.com

	Zalomení
		Version: 1.4.7
		Author: Honza Skypala


== Elements Usage ==
	
	archive : 1
		posts-extra : 1
		theme-archive-title : 1
	footer : 1
		image : 1
		nav-menu : 1
		social-icons : 1
	section : 1
		heading : 1
		posts-extra : 1
	single : 1
		heading : 1
		post-info : 2
		posts-extra : 1
		share-buttons : 2
		theme-post-content : 1
		theme-post-title : 1
	wp-post : 0
		button : 8
		heading : 10
		image : 1
		image-carousel : 1
		posts-extra : 14
		shortcode : 2
		social-icons : 2
		text-editor : 16
	wp-page : 10
		gallery : 1
		gallery-slider : 1
		heading : 7
		image : 3
		image-gallery : 1
		media-carousel : 1
		posts-extra : 1
		text-editor : 6


== Log ==
	
Log: showing 20 of 322020-01-04 13:13:16 [info] Elementor/Upgrades - _v_2_8_3_recalc_usage_data Finished 
2020-01-04 13:13:16 [info] Elementor data updater process has been completed. [array (
  'plugin' => 'Elementor',
  'from' => '2.8.2',
  'to' => '2.8.3',
)]
2020-01-04 13:14:08 [info] Elementor data updater process has been completed. [array (
  'plugin' => 'Elementor Pro',
  'from' => '2.8.2',
  'to' => '2.8.3',
)]
2020-01-23 00:53:35 [info] Elementor data updater process has been completed. [array (
  'plugin' => 'Elementor',
  'from' => '2.8.3',
  'to' => '2.8.4',
)]
2020-01-28 17:16:16 [info] Elementor data updater process has been completed. [array (
  'plugin' => 'Elementor',
  'from' => '2.8.4',
  'to' => '2.8.5',
)]
2020-01-28 17:20:36 [info] Elementor data updater process has been completed. [array (
  'plugin' => 'Elementor',
  'from' => '2.8.5',
  'to' => '2.9.0-beta1',
)]
2020-01-29 19:11:19 [info] Elementor data updater process has been completed. [array (
  'plugin' => 'Elementor',
  'from' => '2.9.0-beta1',
  'to' => '2.9.0-beta2',
)]
2020-02-03 18:55:38 [info] Elementor data updater process has been completed. [array (
  'plugin' => 'Elementor',
  'from' => '2.9.0-beta2',
  'to' => '2.9.0-beta3',
)]
2020-02-07 00:35:29 [info] Elementor data updater process has been completed. [array (
  'plugin' => 'Elementor',
  'from' => '2.9.0-beta3',
  'to' => '2.9.0-beta4',
)]
2020-02-11 13:06:00 [info] Elementor data updater process has been completed. [array (
  'plugin' => 'Elementor',
  'from' => '2.9.0-beta4',
  'to' => '2.9.0-beta5',
)]
2020-02-16 20:40:05 [info] Elementor data updater process has been completed. [array (
  'plugin' => 'Elementor',
  'from' => '2.9.0-beta5',
  'to' => '2.9.2',
)]
2020-02-16 20:42:32 [info] Elementor data updater process has been completed. [array (
  'plugin' => 'Elementor Pro',
  'from' => '2.8.3',
  'to' => '2.8.4',
)]
2020-03-01 23:57:09 [info] Elementor data updater process has been completed. [array (
  'plugin' => 'Elementor',
  'from' => '2.9.2',
  'to' => '2.9.3',
)]
2020-03-10 22:04:07 [info] Elementor data updater process has been completed. [array (
  'plugin' => 'Elementor',
  'from' => '2.9.3',
  'to' => '2.9.5',
)]
2020-03-10 22:07:06 [info] Elementor data updater process has been completed. [array (
  'plugin' => 'Elementor Pro',
  'from' => '2.8.4',
  'to' => '2.9.0-beta2',
)]
2020-03-16 20:26:31 [info] Elementor data updater process has been completed. [array (
  'plugin' => 'Elementor',
  'from' => '2.9.5',
  'to' => '2.9.6',
)]
2020-03-16 20:33:18 [info] Elementor data updater process has been completed. [array (
  'plugin' => 'Elementor Pro',
  'from' => '2.9.0-beta2',
  'to' => '2.9.0-beta3',
)]
2020-03-26 04:06:21 [info] Elementor data updater process has been completed. [array (
  'plugin' => 'Elementor',
  'from' => '2.9.6',
  'to' => '2.9.7',
)]
2020-03-26 04:06:29 [info] Elementor data updater process has been completed. [array (
  'plugin' => 'Elementor Pro',
  'from' => '2.9.0-beta3',
  'to' => '2.9.2',
)]
2020-04-21 16:23:20 [info] Elementor data updater process has been completed. [array (
  'plugin' => 'Elementor Pro',
  'from' => '2.9.2',
  'to' => '2.9.3',
)]

JS: showing 11 of 11JS: 2019-11-03 21:51:26 [error X 2][https://www.blackeye.cz/report/wp-content/plugins/elementor/assets/js/editor.min.js?ver=2.7.5:7:20237] this.ui.sliders.filter(...)[0] is undefined 
JS: 2019-11-03 21:55:41 [error X 1][https://www.blackeye.cz/report/wp-content/plugins/elementor/assets/js/editor.min.js?ver=2.7.5:2:62467] this.ui.tools is undefined 
JS: 2019-11-24 20:03:12 [error X 5][https://www.blackeye.cz/report/wp-content/plugins/elementor/assets/lib/pickr/pickr.min.js?ver=1.4.7:2:4688] t.match is not a function 
JS: 2019-11-24 20:03:26 [error X 10][https://www.blackeye.cz/report/wp-content/plugins/elementor/assets/js/editor.min.js?ver=2.8.0-beta1:30:19534] this.colorPicker is undefined 
JS: 2019-11-25 02:59:12 [error X 1][https://www.blackeye.cz/report/wp-includes/js/jquery/jquery.js?ver=1.12.4-wp:2:12681] Syntax error, unrecognized expression: . 
JS: 2019-12-01 22:00:06 [error X 1][https://www.blackeye.cz/report/wp-content/plugins/elementor/assets/js/editor.min.js?ver=2.8.0-beta2:31:41047] e is null 
JS: 2019-12-13 14:41:08 [error X 2][https://www.blackeye.cz/report/wp-content/plugins/elementor/assets/js/editor.min.js?ver=2.8.1:31:36402] n is undefined 
JS: 2020-01-28 16:21:57 [error X 1][https://www.blackeye.cz/report/wp-content/plugins/elementor/assets/js/common.min.js?ver=2.9.0-beta1:2:104190] Routes: `panel/elements/categories` not found. 
JS: 2020-02-03 17:55:49 [error X 12][https://www.blackeye.cz/report/wp-content/plugins/elementor/assets/js/editor.min.js?ver=2.9.0-beta3:31:82407] this.getEnabledSchemesTypes(...) is undefined 
JS: 2020-02-11 12:07:32 [error X 1][https://www.blackeye.cz/report/wp-content/plugins/elementor/assets/lib/dialog/dialog.min.js?ver=4.7.5:3:4591] this.isVisible is not a function 
JS: 2020-02-12 10:41:08 [error X 1][https://www.blackeye.cz/report/wp-content/plugins/elementor/assets/js/editor.min.js?ver=2.9.0-beta5:31:189981] Elementor panel doesn\'t have page named \'page_settings\' 
```
</details>

**Comment by black-eye** [comment]
@arielk, thanks for paying the attention to the bug.

Just one more suggestion connected to this one. It might be useful to add the ability to activate the Elementor lightbox per gallery instance. It's definitely not necessary, I'm not even sure, if it wouldn't bring more inconsistency into the workflow (I could definitely imagine unskilled editors messing things up), but I'm mentioning it because this option is present in Classic Basic Gallery widget:

![obrazek](https://user-images.githubusercontent.com/5634870/80318906-a60c5580-880d-11ea-9015-2746c86ec3da.png)

It'd be also nice to have the option to turn the lightbox off completely.

Actually, when I first realized the Gallery Pro uses new lightbox, I thought I hade turned this option on by mistake. But then I realized there was no such function there.

**Comment by nicholaszein** [comment]
Hello, @black-eye!

We have great news! 🙌 

### 📢 We're happy to announce that the issue you raised was resolved in Elementor Pro v3.14.0! 🥳 

✅ Feel free to check it out and update your plugin to the new version!

> _Check the changelog for more information:_
> [Changelog of Elementor and Elementor Pro](https://elementor.com/help/changelog-of-elementor/)

Cheers 🥂
